signer-service/src/api/controllers/subsidize.controller.js

```diff
@@ -49,7 +49,7 @@ exports.subsidizePostSwap = async (req, res) => {
     const assetIssuerHex = `0x${Keypair.fromPublicKey(assetIssuer).rawPublicKey().toString('hex')}`;
     const pendulumCurrencyId = {
       Stellar: {
-        AlphaNum4: { code: assetCode.padEnd(4, '\0'), issuer: assetIssuerHex },
+        AlphaNum4: { code: assetCodeRaw.padEnd(4, '\0'), issuer: assetIssuerHex },
```

Comment:
Isn't the idea that we don't need `.padEnd(4, '\0')` for `assetCodeRaw`?

Reply:
Yes you're right here, should remove the pad (which should have never been needed, we were already padding on the config).